apiserver/raft.go

```diff
@@ -65,6 +64,7 @@ func (m *raftMediator) ApplyLease(ctx context.Context, cmd []byte) error {
 
 	m.queue.Enqueue(queue.Operation{
 		Command: cmd,
+		Stop:    ctx.Done(),
```

Comment:
Is `ctx.Done()` guaranteed to always return the same channel? Curious because the typical use in `select` blocks is:

```go
select {
  case <- ctx.Done():
  default:
   //...
}
```

Reply:
Perhaps Stop should be a `StopCh() chan <- struct{}` func instead?